### DISCUSSION TITLE
[SRBench 2024] GPU Support?

### DISCUSSION THREAD

**Discussion by ZakBastiani** [cavalab/srbench#185] [General]
Will there be any support for GPU resources in the 2024 benchmark? Will all of the methods be ran exclusively on CPUs?

**Comment by folivetti** [comment]
Not in this iteration. We're trying to figure it out how to make a fair comparison of CPU-based and GPU-based algorithms

**Reply by folivetti** [reply]
Having said that, suggestions are welcome 😀
